### PR TITLE
Wb 3445 stability tests

### PR DESCRIPTION
**Pull request by vartanianmh**

### CODE REVIEW

File: pipeline/run_stability_tests.sh [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+cat << EOF > $BC_INI
+    {
+        "databases" :
+        [
+            {
+                "key" : "nr",
+                "worker_location" : "/tmp/blast/db",
+                "source_location" : "gs://nr_50mb_chunks",
+                "extensions" : [ "psq", "pin", "phr" ]
```
Comment (christiam) [inline pipeline/run_stability_tests.sh]: Please replace phr with pax

File: pipeline/run_stability_tests.sh [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            },
+            {
+                "key" : "nt",
+                "worker_location" : "/tmp/blast/db",
+                "source_location" : "gs://nt_50mb_chunks",
+                "extensions" : [ "nsq", "nin", "nhr" ]
```
Comment (christiam) [inline pipeline/run_stability_tests.sh]: Please replace nhr with nax

**Review by christiam — CHANGES_REQUESTED** [review]
Mike,
    Could you please save pipeline/asn1.txt.wc.expected in a bucket?

**Comment by vartanianmh** [comment]
> Mike,
> Could you please save pipeline/asn1.txt.wc.expected in a bucket?
File isn't large (318KB) and would like to keep it versioned as we improve our test comparisons.

**Comment by boratyng** [comment]
If we treat asn1.txt.wc.expected as a reference, I think it is fine to have it versioned.

**Review by christiam — APPROVED** [review]

**Pull request merged**